packages/core-web/src/styles/markbind.css

```
@@ -404,23 +404,26 @@ pre {
     position: relative;
 }
 
+/* Override user-agent stylesheets */
+#copy-btn--ua {
+    border: none;
+}
+
 .copy-btn {
-    background-color: #b4b4b9;
-    border-radius: 0.25rem;
-    color: #f8f8ff;
+    background-color: transparent;
+    color: darkgray;
     display: inline-block;
-    font-size: 75%;
-    line-height: 1;
-    padding: 0.25em 0.4em;
+    padding: 0.4rem 0.4rem;
```

```suggestion
    padding: 0.35rem;
```

`0.4` seems a little large, may just be me though =P

leave this one up to you